packages/config/config/devices/0x010f/templates/fgs-2x4.json

```diff
@@ -24,6 +17,7 @@
 			"maxNodes": 5
 		},
 		"3": {
+			"$if": "productType == 0x0204",
```

Comment:
Does it even make sense to template this if all the params are just defined for one device?